packages/components/src/Page/Page.tsx

```diff
@@ -23,14 +23,11 @@ interface PageFoundationProps {
 
   /**
    * Title of the page.
+   *
+   * Supports any React node. If a string is provided, it will be rendered as an H1 heading.
+   * Otherwise it will be rendered as is.
```

Comment:
the reason I mention the IDE JSDocs is because I was thinking it would be good to have a very strong suggestion here detailing that although yes you can technically pass in whatever you want for accessibility and semantic reasons a Page must have exactly one h1 element

technically you could satisfy this requirement elsewhere, with an h1 in some other place but part of the value of Page is that when you go from page A to B to C, they all look pretty familiar so I'd advise against doing that